### ISSUE TITLE
Sphinx 5.0+ prints double colons for roles

### ISSUE THREAD

**Issue by xqt** [sphinx-doc/sphinx#10594]
### Describe the bug

For examle:
:1: Foo
is renderend as
**1:** Foo
with Sphinx 4.5
https://doc.wikimedia.org/pywikibot/stable/library_usage.html

whereas Sphinx 5.0+ renders
**1::** Foo

https://doc.wikimedia.org/pywikibot/master/library_usage.html

### How to Reproduce

Other examples with variables and parameters
Sphinx 4.5 (single quotes after Variables, Parameters, Return type) like: 
**Variables: summary_key** - Edit summary message key
https://doc.wikimedia.org/pywikibot/stable/scripts_ref/scripts.html#module-scripts.basic

Sphinx 5.02  (doible quotes after Variables, Parameters, Return type) like:
**Variables:: summary_key** - Edit summary message key
https://doc.wikimedia.org/pywikibot/master/scripts_ref/scripts.html#module-scripts.basic

### Expected behavior

Single quotes are expected as in Sphinx 4.5

### Your project

See "Show Source" of the given links above.

### Screenshots

Sphinx 4.5
![image](https://user-images.githubusercontent.com/1149488/175325484-751bb32e-5a7b-48ad-87c7-622140fc2cf4.png)

Sphinx 5.0.2
![image](https://user-images.githubusercontent.com/1149488/175325849-3f2ef4cc-efd6-4584-b553-f4748acce136.png)

### OS

Linux

### Python version

Python 3.7

### Sphinx version

4.5.0 vs. 5.0.2

### Sphinx extensions

'sphinx.ext.autodoc', 'sphinx.ext.autosectionlabel', 'sphinx.ext.extlinks',     'sphinx.ext.todo',     'sphinx.ext.viewcode',     'sphinx.ext.autosummary',     'sphinx.ext.napoleon',

**Comment by jfbu** [comment]
This is not a Sphinx change per se, I can reproduce it with Sphinx 4.2.0, but is impacted by Docutils `0.18`.

Try ``pip install docutils==0.17.1``.

Minimal example:
```rest
Welcome to FOO's documentation!
===============================

test
----

:1: hello

:2: world
```

**Comment by jfbu** [comment]
From Docutils [0.18 release notes](https://docutils.sourceforge.io/RELEASE-NOTES.html#release-0-18-2021-10-26):
> Write footnote brackets and field term colons to HTML, so that they are present also without CSS and when copying text.

Our `basic.css` does
```css
dl.field-list > dt:after {
    content: ":";
}
```
but there is already since DocUtils 0.18 a ``<span class="colon">:</span>``.

Adding
```
dl.field-list  .colon {
    display: none;
}
```
to `basic.css` appears to fix the problem in a way which should not depend on Docutils version.  Assuming a `colon` class was not used by extensions etc....

ping @AA-Turner for advice. Is there a potential issue with footnote brackets? I have not followed episodes of Sphinx compatibitility patches for Docutils 0.18.

**Comment by AA-Turner** [comment]
Fixed by #10595 

A
